spec/ffprober/parser_spec.rb

```diff
@@ -1,109 +1,82 @@
 
-    it 'should determine the correct sample rate of the first audio stream' do
-      expect(ffprobe.audio_streams.first.sample_rate).to eq('44100')
-    end
-  end
-
-  describe 'subtitle_streams' do
-    it 'should determine the correct duration_ts of the first subtitle stream' do
+      expect(ffprobe.subtitle_streams.count).to eq(1)
       expect(ffprobe.subtitle_streams.first.duration_ts).to eq(98)
-    end
-
-    it 'should determine the correct bitrate of the first subtitle stream' do
-      expect(ffprobe.subtitle_streams.first.bit_rate).to eq('5690')
-    end
-
-    it 'should determine the correct language of the first subtitle stream' do
+      expect(ffprobe.subtitle_streams.first.bit_rate).to eq("5690")
       expect(ffprobe.subtitle_streams.first.tags[:language]).to eq("eng")
-    end
 
-    it 'should determine the correct number of subtitle streams' do
-      expect(ffprobe.subtitle_streams.count).to eq(1)
-    end
-  end
-
-  describe 'video_streams' do
-    it 'should determine the correct width of the first video streams' do
       expect(ffprobe.video_streams.first.width).to eq(480)
-    end
-
-    it 'should determine the correct width of the first video streams' do
       expect(ffprobe.video_streams.first.height).to eq(272)
-    end
-
-    it 'should determine the correct number of video streams' do
       expect(ffprobe.video_streams.count).to eq(1)
-    end
-  end
 
-  describe 'chapters' do
-    it 'should contain the chapters' do
       expect(ffprobe.chapters.count).to eq(3)
-    end
-
-    describe 'chapter' do
-      subject(:chapter) { ffprobe.chapters.first }
-
-      it 'should contain id' do expect(chapter.id).to eq(0) end
-      it 'should contain time_base' do expect(chapter.time_base).to eq('1/1000') end
-      it 'should contain start' do expect(chapter.start).to eq(0) end
-      it 'should contain start_time' do expect(chapter.start_time).to eq('0.000000') end
-      it 'should contain end' do expect(chapter.end).to eq(10) end
-      it 'should contain end_time' do expect(chapter.end_time).to eq('0.010000') end
-      it 'should contain tags' do expect(chapter.tags).to eq(title: 'Chapter 1') end
+      expect(ffprobe.chapters.first.id).to eq(0)
+      expect(ffprobe.chapters.first.time_base).to eq("1/1000")
+      expect(ffprobe.chapters.first.start).to eq(0)
+      expect(ffprobe.chapters.first.start_time).to eq("0.000000")
+      expect(ffprobe.chapters.first.end).to eq(10)
+      expect(ffprobe.chapters.first.end_time).to eq("0.010000")
+      expect(ffprobe.chapters.first.tags).to eq(title: "Chapter 1")
     end
   end
-end
 
-describe Ffprober::Parser do
+  context "from_json" do
+    let(:ffprobe) { Ffprober::Parser.from_json(File.read("spec/assets/sample video.json")) }
 
-  context 'from_file', if: Ffprober::FfprobeVersion.valid? do
-    let(:ffprobe) { Ffprober::Parser.from_file('spec/assets/sample video.m4v') }
+    it 'parses the content' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.